cmd/internal/shared/config.go

```diff
@@ -48,8 +48,9 @@ var DefaultClusterConfig = cluster.Config{}
 
 // DefaultHTTPConfig is the default HTTP config.
 var DefaultHTTPConfig = config.HTTP{
-	Listen:    ":1885",
-	ListenTLS: ":8885",
+	Listen:         ":1885",
+	ListenTLS:      ":8885",
+	TrustedProxies: []string{"10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"},
```

Comment:
Shouldn't this be explicitly specified by the user? Or at least the user should be able to overwrite these. Currently, it seems that the only option is append.

Reply:
The goal of the default configuration is to have some defaults that work well for most deployments. These three CIDRs match the private address space specified in RFC1918. Setting this config explicitly in config files, environment or flags will overwrite it, not append.